Sources/StreamChatUI/MessageActionsPopup/ChatMessageActionsVC.swift

```diff
@@ -132,7 +132,7 @@ open class ChatMessageActionsVC: _ViewController, ThemeProvider {
                 self.alertsRouter.showMessageDeletionConfirmationAlert { confirmed in
                     guard confirmed else { return }
 
-                    self.messageController.deleteMessage { _ in
+                    self.messageController.deleteMessage() { _ in
```

Comment:
probably a good idea to add `hard` to the `messageController.deleteMessage` method